lib/media/segment_prefetch.js

```diff
@@ -117,6 +127,11 @@ shaka.media.SegmentPrefetch = class {
       this.iterator_.next();
     }
     const promises = [];
+    let prefetchLimit = this.prefetchLimit_;
+    const rate = Math.abs(this.getPlaybackRate_());
+    if (rate > 1) {
+      prefetchLimit += (Math.round(rate) - 1);
```

Comment:
Wouldn't the playback rate be a multiplier to the number of segments?

Reply:
If you prefer it, I can change it. What do you prefer?

Reply:
Done!